hail/src/test/scala/is/hail/expr/ir/RandomFunctionsSuite.scala

```diff
@@ -51,7 +51,7 @@ class RandomFunctionsSuite extends HailSuite {
 
   implicit val execStrats = ExecStrategy.javaOnly
 
-  val counter = ApplySeeded("counter_seeded", FastSeq(), 0L, TInt32())
+  def counter = ApplySeeded("counter_seeded", FastSeq(), 0L, TInt32())
```

Comment:
@catoverdrive This fixes the bug. The problem is that Interpret's function memoization is doing the wrong thing when seeded function IRs appear with the same reference in the IR tree. We could deep copy going into interpret, I suppose?